chore(guide): Several backend cleanup and fixes (#2378)
TRaSH-Guides/Guides: CONTRIBUTING.md

👍🎉 First off, thanks for taking the time to contribute! 🎉👍

Here you will find guidelines for contributing to [TRaSH Guides](https://trash-guides.info/). These are mostly guidelines, not rules. Use your best judgment and feel free to propose changes to this document in a pull request.

## Guidelines

- Ensure you're editing the latest version.

> - If you suggest changes, ensure they don't disrupt anything else in the guides.

Reviewer: ```suggestion
- If you suggest changes, ensure they don't cause any issues, inconsistencies or breakages elsewhere in the guides.
```

- After the modifications, verify that you do not receive any [markdownlint](https://github.com/markdownlint/markdownlint/blob/master/docs/RULES.md) errors.

> - If the `metadata.json` and `metadata.schema.json` require changes, announce it first in the Discord `#development` channel and ping `voidpointer` and `nitsua` to prevent disruption in third-party apps :bangbang:

Reviewer: ```suggestion
- If the `metadata.json` and `metadata.schema.json` require changes, announce it first in the Discord `#development` channel and ping `voidpointer` and `nitsua` to prevent disruption to third-party apps :bangbang:
```

> [!CAUTION]
>

> > - The name of the Custom Format must match the JSON name or the other way around. :bangbang:

Reviewer: ```suggestion
> - The names of the Custom Format and the JSON must be identical. :bangbang:
```

### File Naming

> [!CAUTION]

> > JSON file names are always written in lowercase: spaces are replaced by dashes, the `+` sign is replaced by `plus`, and no spaces or special characters exist except for a dash. :bangbang:

Reviewer: ```suggestion
> JSON file names are always written in lowercase. Spaces must be replaced by dashes, `plus` should be used instead of the `+` symbol, and no spaces or special characters should be used except for dashes. :bangbang:
```

## YAML file naming

> - YAML file names are always written in lowercase: spaces are replaced by dashes, the `+` sign is replaced by `plus`, and no spaces or special characters exist except for a dash. :bangbang:

Reviewer: ```suggestion
- YAML file names are always written in lowercase. Spaces must be replaced by dashes, `plus` should be used instead of the `+` symbol, and no spaces or special characters should be used except for dashes. :bangbang:
```
